src/TopicFootball.jsx

```diff
@@ -1,82 +1,78 @@
-import { useEffect, useState } from "react";
-import { fetchArticles, fetchTopics } from "./api";
-import ArticleCard from "./ArticleCard";
-import { Link, useSearchParams } from "react-router-dom";
+// import { useEffect, useState } from "react";
+// import { fetchArticles, fetchTopics } from "./api";
+// import ArticleCard from "./ArticleCard";
+// import { Link, useSearchParams } from "react-router-dom";
 
 export default function TopicFootball() {
```

Comment:
I'd go ahead and delete this as it seems like you've got the topics and the sort working